maintenance/pre-commit-check.sh

```diff
@@ -0,0 +1,8 @@
+#! /usr/bin/env bash
+
+set -e
+
+shfmt -bn -i 4 -d helpers/helpers
+shfmt -bn -i 4 -d helpers/helpers.v2.1.d/*
+shfmt -bn -i 4 -d helpers/helpers.v2.d/*
+shfmt -bn -i 4 -d hooks/*
```

Comment:
```suggestion
shfmt -sr -bn -i 4 -d helpers/helpers
shfmt -sr -bn -i 4 -d helpers/helpers.v2.1.d/*
shfmt -sr -bn -i 4 -d helpers/helpers.v2.d/*
shfmt -sr -bn -i 4 -d hooks/*
```
Add `-sr`. This option adds a space after file redirection carrets, so it creates for example this diff:

```
-    _generate_config >${pending_dir}/etc/yunohost/mdns.yml
+    _generate_config > ${pending_dir}/etc/yunohost/mdns.yml
```